README.md

```diff
@@ -24,15 +24,20 @@ Backend server code for Baot mentorship application.
    versioning conflicts.
 
    > :mega: If you're experiencing installation problems with _psycopg2_,
-   > try installing it separately using the command `pip install psycopg2-binary`.
+   > try installing it separately using the command `pip install psycopg2-binary`. (in such case don't forget to comment it on the requirements.txt file and re-run `pip install -r requirements.txt`)
 
 2. Set the following environment variables (you can do so in PyCharm's run configurations):
    - `DATABASE_URL=postgresql://postgres:somePassword@localhost/postgres`
    - `SECRET_KEY=someSecretStringUsedByFlask`
    - `APP_SETTINGS=config.DevelopmentConfig`
-   - `GOOGLE_APPLICATION_CREDENTIALS=secretKey` - for prod use FIREBASE_CONFIG instead
+   - `GOOGLE_APPLICATION_CREDENTIALS=secretKey.json` - for prod use FIREBASE_CONFIG instead
 3. Run `python reset_db.py` to initialize the DB tables.
```

Comment:
I think the change here should be GOOGLE_APPLICATION_CREDENTIALS=<path to secret key> because this is the intension, and anyone can call and put her secret key in different ways. WDYT?